File: spring/src/main/java/com/example/spring/model/Olympian.java
package com.example.spring.model;

import javax.persistence.Entity;
import javax.persistence.Table;
import javax.persistence.Column;
import javax.persistence.Id;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

@Entity
@Table(name = "olympians")
public class Olympian {
  @Id
  @Column(name = "id")
  Long id;

  @Column(name="name")
  String name;

  @Column(name="sex")
  String sex;

  @Column(name="age")
  Integer age;

  @Column(name="height")
  Integer height;

  @Column(name="weight")
  Integer weight;
}

File: phoenix/lib/phoenix_benchmark/olympians/olympian.ex
defmodule PhoenixBenchmark.Olympians.Olympian do
	use Ecto.Schema
	
	schema "olympians" do
		field :name, :string
		field :sex, :string
		field :age, :integer
		field :height, :integer
		field :weight, :integer
	end
end
